### PR TITLE
update APTC calculation to incorporate pediatric dental SLCSP

### PR DESCRIPTION
**Pull request by nisanthyaganti9**
ME-180199876

### CODE REVIEW

**Review by markgoho — CHANGES_REQUESTED** [review]
please add ticket reference

File: components/benefit_markets/app/models/benefit_markets/products/product.rb
Comment (markgoho) [inline components/benefit_markets/app/models/benefit_markets/products/product.rb:377]: Is there an enum we could use instead of `Covered`?

Comment (nisanthyaganti9) [reply components/benefit_markets/app/models/benefit_markets/products/product.rb:377]: hmm, there isn't. But, I can add one i guess. I will add it if i have to make another commit.

**Comment by nisanthyaganti9** [comment]
Need reload plans & rates, reset cache after merging this PR.

**Review by markgoho — DISMISSED** [review]
thanks @nisanthyaganti9 -- I can't approve this for merging, but I can confirm that you've made the changes I requested 👍🏻

**Review by vkghub — DISMISSED** [review]
changes looks good

**Review by vkghub — DISMISSED** [review]

**Review by markgoho — CHANGES_REQUESTED** [review]
request that this be deployed for functional testing before merging

**Comment by markgoho** [comment]
closing this while we re-evaluate the strategy to deliver this feature

**Pull request closed without merging**
